sass/atoms/_links.scss

```diff
@@ -14,6 +14,8 @@ a {
   }
 
   &.external {
+    display: inline-block;
+    
```

Comment:
just a small nit picked up by Prettier. Seems like here are 4 spaces on this line:

```suggestion

```

Reply:
Going to go ahead and merge the commit suggestions.

Reply:
Thanks!